src/main/react-webapp/src/components/Example.js

```diff
@@ -0,0 +1,25 @@
+import {Link} from 'react-router-dom'
+import React from "react";
+
+const Example = () => {
+
+    // Just an example component to show how routing to another component/page should work.
+
+    const centerDiv = {
+        position: "absolute",
+        width: "100%",
+        top: "50%",
+        textAlign: "center",
+    }
+    const text = {
+        fontSize: "xx-large",
+    }
+
+    return(
+        <div style={centerDiv}>
```

Comment:
I realize this is just an example, but I have a fairly strong preference for avoiding inline styling and keeping styles in a separate CSS file or CSS module.

Reply:
yeah I agree. I just wanted to quickly center the buttons